frontend/src/components/base/Calendar.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../base.scss';
+
+.calendar {
+    background-color:white;
+    align-self: center;
+}
+.calendar__header {
+    margin: 20px 0 50px 0px;
+    text-align: center;
+    vertical-align: middle;
+}
+.calendar__body {
+    display: flex;
+}
+.calendar__table {
+    height: 300px;
+    width: 500px;
+    margin: 0 20px 0 20px;
+}
```

Comment:
@cw10108 I think it would be nicer that header, body, table are located in .calendar style to show hierarchy.

```scss
.calendar {
    background-color:white;
    align-self: center;

.calendar__header {
    margin: 20px 0 50px 0px;
    text-align: center;
    vertical-align: middle;
}

.calendar__body {
    display: flex;
}
.calendar__table {
    height: 300px;
    width: 500px;
    margin: 0 20px 0 20px;
}

}
```